websites/H/HackTheBox/metadata.json

```
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "937757295453044806",
+		"name": "Atsukoro1#6323"
```

Remove discriminator from old `pmd`
```suggestion
		"name": "Atsukoro1"
```